core/kits/assets/js/panel-menu.js

```diff
@@ -13,14 +13,22 @@ PanelMenu.createGroupItems = ( groupName, keys ) => {
 
 	return keys.map( ( key ) => {
 		const fullKey = groupName + '-' + key,
-			tab = tabs[ fullKey ];
-
-		return {
-			name: fullKey,
-			icon: tab.icon,
-			title: tab.title,
-			callback: () => $e.route( 'panel/global/' + fullKey ),
-		};
+			tab = tabs[ fullKey ],
+			item = {
+				name: fullKey,
+				icon: tab.icon,
+				title: tab.title,
+			};
+
+		if ( 'additional-settings' === key ) {
```

Comment:
Instead of doing it inside the general `createGroupItems` method, do it where you create the "settings" group